packages/ui-toolkit/.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": ["joyent-portal"],
   "plugins": [
+    "lodash",
```

Comment:
https://github.com/yldio/babel-preset-joyent-portal/blob/master/index.js#L3 we're already doing this ;)